expenses/reports.py

```diff
@@ -149,13 +185,42 @@ def tabulate(self, results: typing.Iterable, engine: Engine) -> SafeString:
             )
         )
 
-    def run(self) -> SafeString:
+    def create_file(self, results: typing.Iterable, engine: Engine) -> HttpResponse:
+        column_headers: (typing.List[str], typing.List[str]) = self.get_column_headers(engine, False)
+        column_header_names = column_headers[0]
+
+        first_row, results = peek(self.preprocess_rows(results, False))
+
+        response = HttpResponse(content_type="text/csv")
+        response["Content-Disposition"] = 'attachment; filename="report.csv"'
```

Comment:
Or perhaps we could name the file with the report type and current date? (Not necessary, just a bonus suggestion.)